From 7566e3e433bd6ea44785b08164506fd013fe2b51 Mon Sep 17 00:00:00 2001
From: "Richard W.M. Jones" <rjones@redhat.com>
Date: Tue, 8 Jul 2025 11:11:48 +0100
Subject: [PATCH] docs: Move -oo verify-server-certificate docs to alphabetical
 place

This option wasn't in alphabetical order.

(cherry picked from commit 378967fc4984af4f06c586886a1148aebede8cff)
---
 docs/virt-v2v.pod | 16 ++++++++--------
 1 file changed, 8 insertions(+), 8 deletions(-)

diff --git a/docs/virt-v2v.pod b/docs/virt-v2v.pod
index 3cfb0948..3eefe404 100644
--- a/docs/virt-v2v.pod
+++ b/docs/virt-v2v.pod
@@ -611,14 +611,6 @@ volume property.
 When using I<-o qemu> only, this boots the guest immediately after
 virt-v2v finishes.
 
-=item B<-oo verify-server-certificate>
-
-=item B<-oo verify-server-certificate=>C<true|false>
-
-For I<-o openstack> (L<virt-v2v-output-openstack(1)>) only, this can
-be used to disable SSL certification validation when connecting to
-OpenStack by specifying I<-oo verify-server-certificate=false>.
-
 =item B<-oo os->*B<=>*
 
 For I<-o openstack> (L<virt-v2v-output-openstack(1)>) only, set optional
@@ -736,6 +728,14 @@ The OVF format understood by oVirt REST API.
 For backward compatibility the default is I<ovirtexp>, but this may change in
 the future.
 
+=item B<-oo verify-server-certificate>
+
+=item B<-oo verify-server-certificate=>C<true|false>
+
+For I<-o openstack> (L<virt-v2v-output-openstack(1)>) only, this can
+be used to disable SSL certification validation when connecting to
+OpenStack by specifying I<-oo verify-server-certificate=false>.
+
 =item B<-op> file
 
 Supply a file containing a password to be used when connecting to the
